MySQLdb/_mysql.c

```diff
@@ -486,6 +488,21 @@ _mysql_ConnectionObject_Initialize(
     }
     Py_BEGIN_ALLOW_THREADS ;
     self->open = 1;
+
+    #ifdef MARIADB_BASE_VERSION
+    if (ssl_mode) {
+        if (strcasecmp(ssl_mode, "PREFERRED") != 0)
+        {
+            int enforce_tls= 0;
+            if (strcasecmp(ssl_mode, "REQUIRED") == 0)
```

Comment:
> So this option without VERIFY_CA or VERIFY_INDENTITY flags is pretty much useless. Since it will still work in opportunistic mode.

Yep, but looking from the user perspective and making it work like MySQL means that in `REQUIRED` means TLS enforced, but no certificate validation. So in that case, `MYSQL_OPT_SSL_VERIFY_SERVER_CERT` should be set. 

If `MYSQL_OPT_SSL_VERIFY_SERVER_CERT` does get set, it means `REQUIRED` on MariaDB is equivalent to `VERIFY_IDENTITY` on MySQL which seems very confusing?